src/background.js

```diff
@@ -6,18 +6,15 @@ const activeTabUrlMemo = {}
  * To support Github SPA
  * ref: https://developer.chrome.com/docs/extensions/reference/webNavigation/
  */
-chrome.webNavigation.onHistoryStateUpdated.addListener(
-  ({ tabId, url }) => {
-    if (activeTabUrlMemo[tabId] !== url) {
-      activeTabUrlMemo[tabId] = url
+chrome.webNavigation.onHistoryStateUpdated.addListener(({ tabId, url }) => {
+  if (activeTabUrlMemo[tabId] !== url) {
+    activeTabUrlMemo[tabId] = url
 
-      chrome.tabs.sendMessage(tabId, {
-        type: GLOBAL_MESSAGE_TYPE.ON_HISTORY_UPDATED,
-      })
-    }
-  },
-  { url: [{ hostSuffix: 'github.com' }] }
-)
+    chrome.tabs.sendMessage(tabId, {
```

Comment:
Not sure should we stop sending a custom message if the URL is not GitHub or Github Enterprise?

Reply:
Sure we need, I've opened a ticket for it